app/next-client-app/next.config.js

```diff
@@ -1,6 +1,11 @@
 /** @type {import('next').NextConfig} */
 const nextConfig = {
   output: "standalone",
+  experimental: {
+    serverActions: {
+      allowedOrigins: ["127.0.0.1:7000"],
```

Comment:
We want this to be an environment variable. It can use the existing backend_url one
```suggestion
      allowedOrigins: [process.env.BACKEND_URL],
```